app/Offer/OfferServiceProvider.php

```diff
@@ -562,6 +566,14 @@ public function register(): void
             fn () => new UpdateWorkflowStatusRequestHandler($container->get('event_command_bus'))
         );
 
+        $container->addShared(
+            ManageContributorsRequestHandler::class,
+            fn () => new ManageContributorsRequestHandler(
+                $container->get(OfferRepository::class),
+                $container->get(ContributorRepository::class)
```

Comment:
If you use the `ContributorRepository` interface name instead in the service container (with suffix at the moment, or without if you remove it like I suggested in the other comment) the service container should be able to auto-wire `ManageContributorsRequestHandler` because it knows both the required `OfferRepository` and the required `ContributorRepository(Interface)`. So you should be able to leave out this explicit `$container->addShared(ManageContributorsRequestHandler::classaddShared, ...)` then.